docs/reference/build-web-app.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Build a web app with ONNX Runtime
+description: Considerations and options for building a web application with ONNX Runtime
+parent: Reference
+has_children: false
+nav_order: 5
+---
+
+# Build a web application with ONNX Runtime
+{: .no_toc}
+
+## Contents
+{: .no_toc}
+
+* TOC
+{:toc}
+
+This document explains the options and considerations for building a web application with ONNX Runtime.
+
+## Options for deployment target
+
+1. Inference in browser
+
+   Runtime and model are downloaded to client and inferencing happens inside browser. Use onnxruntime-web in this scenario.
+
+2. Inference on server
+
+   Browser sends user's input to server, server inferences and gets the result and sends back to client.
+
+   Use native ONNX Runtime to get best performance.
+
+   To use Node.js as the server application, use onnxruntime-node (ONNX Runtime node.js binding) on the server.
+
+3. Electron
+
+   Electron uses a frontend (based on chromium, technically a browser core) and a backend (based on Node.js). Examples are VSCode, Teams before 2021.
```

Comment:
remove the examples (or at least remove Teams)

Reply:
Removed